### PR TITLE
Droth 740 bus stops not attached to road links

### PR DESCRIPTION
**Pull request by MAC-CGI**
Droth-740 bus stops not attached to road links

### CODE REVIEW

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/MassTransitStopService.scala [hunk 1]
```diff
@@ -73,6 +75,27 @@ trait MassTransitStopService extends PointAssetOperations {
...
+
+  private def createPersistedAsset[T](persistedStop: PersistedAsset, asset: AssetAdjustment) = {
+
+    new PersistedAsset(asset.assetId, persistedStop.nationalId, asset.linkId, persistedStop.stopTypes, persistedStop.municipalityCode, asset.lon, asset.lat,
+      asset.mValue, persistedStop.validityDirection, persistedStop.bearing, persistedStop.validityPeriod, persistedStop.floating, persistedStop.vvhTimeStamp,
+      persistedStop.created, persistedStop.modified, persistedStop.propertyData, persistedStop.linkSource)
+  }
+  private def adjustmentOperation(persistedAsset: PersistedAsset, adjustment: AssetAdjustment): Long = {
+    val position = Some(Position(adjustment.lon, adjustment.lat, adjustment.linkId, persistedAsset.bearing))
+    updateExistingById(persistedAsset.id, position, persistedAsset.propertyData.map(a => SimpleProperty(a.publicId , a.values)).toSet, "a", () => _ , false)
```
Comment (joanacoelho) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/MassTransitStopService.scala]: Probably you should change the username ` "a"` to the current user :smile:

**Comment by joanacoelho** [comment]
Ok

**Comment by marquesfm** [comment]
Ok for me

**Comment by halied** [comment]
Ok

**Pull request merged**